docs/reference/deploy/master-config-reference.rst

```diff
@@ -403,14 +403,21 @@ resource pool ``max_slots_per_pod``.
 ``slot_type``
 -------------
 
-Resource type used for compute tasks. Defaults to ``cuda``.
+Resource type used for compute tasks. Valid options are ``gpu``, ``cuda``, ``cpu``, or ``rocm``.
+Defaults to ``cuda``.
```

Comment:
Does it default to `cuda` or `gpu`? The helm config reference says `gpu`?

Reply:
The master config defaults to ```cuda```, helm defaults to ```gpu```